virt_lightning/virt_lightning.py

```diff
@@ -494,12 +494,13 @@ def load_ssh_key_file(self, ssh_key_file):
             with open(os.path.expanduser(ssh_key_file), "r") as fd:
                 self.ssh_key = fd.read()
         except IOError:
-            raise Exception(
```

Comment:
I think you can handle that exception, and use `logger.error('something going wrong', exc_info=True)` exc_info will print exception to the console output. Anyway that isn't issue just recommendation.

Reply:
Good point, we can also do https://docs.python.org/3/library/logging.html#logging.Logger.exception

This being said, the benefit here with the current code base is that the exception will `exit(1)` for use.